<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCoffeeSupportRunDialog</class>
 <widget class="QDialog" name="TCoffeeSupportRunDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Align with T-Coffee</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Advanced options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="gapOpenCheckBox">
          <property name="text">
           <string>Gap opening penalty</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="gapExtCheckBox">
          <property name="text">
           <string>Gap extension penalty</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="maxNumberIterRefinementCheckBox">
          <property name="text">
           <string>Number of iterations</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="maxNumberIterRefinementSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="gapOpenSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>-10000</number>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="singleStep">
           <number>50</number>
          </property>
          <property name="value">
           <number>-50</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="gapExtSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>-5000</number>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="alignButton">
       <property name="text">
        <string>Align</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>maxNumberIterRefinementCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxNumberIterRefinementSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gapOpenCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gapOpenSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gapExtCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gapExtSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
